src/odemis/driver/npmc.py

```diff
@@ -550,7 +558,7 @@ def _waitEndMove(self, future, axes, end=0):
 
         last_upd = time.time()
         dur = max(0.01, min(end - last_upd, 60))
-        max_dur = dur * 2 + 1
+        max_dur = dur * 5 + 1
```

Comment:
Does `max_dur` still need to be bumped up that much, now that the move duration estimation should be correct? If not, bring it back. If still, please explain (in a comment) why the estimation is still so bad.